src/static/login.html

```diff
@@ -11,7 +11,29 @@
 
         (function(console) {
             var environment = window.environment || { };
+            var oauth = window.oauth || null;
```

Comment:
There's a lot that is shared, esp css, layout and basic java-script. And if we are trying to avoid haveing login.html load any other resources it's hard to share that stuff. But i'm ok with spiting them up if you think that's better.